app/src/main/java/fr/free/nrw/commons/category/CategoryClient.kt

```diff
@@ -16,124 +16,123 @@ const val CATEGORY_NEEDING_CATEGORIES = "needing categories"
  */
 @Singleton
 class CategoryClient
-    @Inject
```

Comment:
try to avoid the changes which are not relevant to the PR(formating etc). This leads to increase in unrelated git diff which becomes harder for reviewers to review the changes.